client/src/pages/course/admin/students.tsx

```diff
@@ -39,6 +39,10 @@ function Page(props: Props) {
 
   const [loading, withLoading] = useLoading(false);
   const [courseManagerRole] = useState(isCourseManager(props.session, courseId));
+  const courseSupervisorRole = useMemo(
```

Comment:
I would suggest to name it like `isCourseManager` + `isCourseSupervisor` or `hasCourseManagerRole` + `hasCourseSupervisorRole`, current naming refers corresponding roles objects rather than flags for me :)

Reply:
fixed